### PR TITLE
Updated terms and conditions

### PR DESCRIPTION
**Pull request by mucow**

### CODE REVIEW

File: src/pages/TermsPage.js [hunk 1]
```diff
@@ -18,29 +18,29 @@ class TermsPage extends React.Component {
...
-                    <p>The service providers listed in the directory do not under any circumstances provide services for or on behalf of Infoxchange.</p>
-                    <p>You agree that, to the fullest extent permitted by applicable law and subject to your Consumer Rights, Infoxchange will not be liable except to the extent due to Infoxchange’s negligence or breach of these terms  for: (i) any expenses, losses, damages, or costs suffered or incurred by you or any other person as a result of the provision or failure by the service providers to provide any services  listed in the directory; (ii) access delays to, access interruptions to, suspension of or discontinuation of the App; (iii) data non-delivery, misdelivery, corruption, destruction or other modification to the data in or through the use of the App; (iv) viruses, system failures or malfunctions which may occur in connection with your use of the App; and (v) any inaccuracies or omissions in content contained in the App.</p>
+                    <p>The service providers listed in the directory do not under any circumstances provide services for or on behalf of Infoxchange.</p>
```
Comment (kane-c) [inline src/pages/TermsPage.js]: There's a trailing space on this line. The linter might not be happy

File: src/pages/TermsPage.js [hunk 2]
```diff
@@ -18,29 +18,29 @@ class TermsPage extends React.Component {
...
+                    <p>The service providers listed in the directory do not under any circumstances provide services for or on behalf of Infoxchange.</p> 
+                    <p>You agree that, to the fullest extent permitted by applicable law and subject to your Consumer Rights, Infoxchange will not be liable except to the extent due to Infoxchange’s negligence or breach of these terms  for: (i) any expenses, losses, damages, or costs suffered or incurred by you or any other person as a result of the provision or failure by the service providers to provide any services  listed in the directory; (ii) access delays to, access interruptions to, suspension of or discontinuation of the App; (iii) data non-delivery, misdelivery, corruption, destruction or other modification to the data in or through the use of the App; (iv) viruses, system failures or malfunctions which may occur in connection with your use of the App; and (v) any inaccuracies or omissions in content contained in the App. </p>
```
Comment (kane-c) [inline src/pages/TermsPage.js]: Suggest removing the space before the closing tag on the end of this line

File: src/pages/TermsPage.js [hunk 3]
```diff
@@ -18,29 +18,29 @@ class TermsPage extends React.Component {
 
                 <div className="body">
                     <p><strong>Ask Izzy Application Terms of Use</strong></p>
-                    <p>Thank you for downloading the Ask Izzy application ( <strong>App</strong> ), which is operated by Infoxchange (ABN 74 457 506 140). Access to, and use of, this App is provided strictly subject to these terms of use. By downloading and using this App, you accept these terms of use and our <a href="https://www.infoxchange.net.au/privacy-policy">Privacy Policy</a> (collectively the <strong>App Terms</strong> ). Please only download and use this App if you have read, understood and agree to these terms of use.</p>
+                    <p>Thank you for accessing the Ask Izzy application (<strong>App</strong>), which is operated by Infoxchange (ABN 74 457 506 140). Access to, and use of, this App is provided strictly subject to these terms of use. By  using this App, you accept these terms of use and our <a href="https://www.infoxchange.net.au/privacy-policy">Privacy Policy</a> (collectively the <strong>App Terms</strong>). Please only use this App if you have read, understood and agree to these terms of use.</p>
```
Comment (kane-c) [inline src/pages/TermsPage.js]: There's a double space between these words: `By  using`. Probably not important :P

**Comment by kane-c** [comment]
++

File: src/pages/TermsPage.js (hunk 2 above)
Comment (mucow) [inline src/pages/TermsPage.js]: Damn i thought i got rid of all those. Stupid copy and paste from ms word.

**Pull request merged**